Analysis/DEEP2_R23_O32/Binning/N_Bins_Grid_Analysis.py

```diff
@@ -162,21 +191,29 @@ def n_times_binned(fitspath, dataset, pdf_pages, outfile, n_split, R23, O32, SNR
         y_average = [yBar[jj], yBar[jj]]
         plt.plot(x_value, y_value, linewidth=0.3, color='b')
         plt.plot(x_value, y_average, linewidth=0.3, color='g')
+
     fig.savefig(pdf_pages, format='pdf')
     pdf_pages.close()
 
-    np.savez(outfile, locator=locator, R23_minimum=R23_minimum, O32_minimum=O32_minimum, Number_inbin=Number_inbin)
-
-    n1 = ('ID', 'R23_minimum', 'O32_minimum', 'xBar', 'yBar', 'R23_median', 'O32_median', 'area')
-    tab1 = Table([n_bins_range, R23_lowlimit, O32_lowlimit, xBar, yBar, R23_medians, O32_medians, area], names=n1)
-    asc.write(tab1, fitspath+'/'+dataset+'_binning_averages.tbl', format='fixed_width_two_line')
+    np.savez(outfile, locator=locator, R23_minimum=R23_minimum, O32_minimum=O32_minimum, Number_inbin=number_inbin)
```

Comment:
You might consider a dictionary approach:

```
avg_dict = {'locator': locator,
            'R23_minimum': R23_minimum,
            'O32_minimum': O32_minimum,
            'Number_inbin': Number_inbin}
```

Then:
```
np.savez(outfile, **avg_dict)

Reply:
done